articles/active-directory/develop/authentication-scenarios.md

```diff
@@ -57,6 +57,25 @@ In the example scenario above, you can classify the apps according to these two
 * Apps that need to securely access resources
 * Apps that play the role of the resource itself
 
+### How each flow emits tokens and codes
+
+Depending on how your client is built, it can use one (or several) of the authentication flows supported by the Microsoft identity platform.  These flows can produce a variety of tokens (id_tokens, refresh tokens, access tokens) as well as authorization codes, and require different tokens to make them work. This chart proides an overview:
+
+|Flow | Requires | id_token | access token | refresh token | authorization code | 
+|-----|----------|----------|--------------|---------------|--------------------|
+|[Authorization code flow](v2-oauth2-auth-code-flow) | | x | x | x | x|  
```

Comment:
@hpsin : the links to the flow pages are all mising a .md.
I can take care of this if you wish;